rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
```

Comment:
Yeah, when you replace it with `gem 'rails', github: 'rails'` you will be able to run these benchmarks locally.